test/integration/controller/jobs/pod/pod_controller_test.go

```diff
@@ -289,12 +289,6 @@ var _ = ginkgo.Describe("Pod controller", ginkgo.Ordered, ginkgo.ContinueOnFailu
 
 					util.ExpectPodUnsuspendedWithNodeSelectors(ctx, k8sClient, lookupKey, map[string]string{"kubernetes.io/arch": "arm64"})
 
-					gomega.Eventually(func(g gomega.Gomega) {
```

Comment:
why did this go away?

Reply:
The condition is flaky , there are cases when the started event is not the last one in the list (but present).  Checking the workload conditions and pod suspended state should be sufficient.

Reply:
We'll need some form of Event integration test back. But we can do that in follow ups.